src/components/side-panel/layer-panel/text-label-panel.js

```diff
@@ -40,10 +39,9 @@ import LayerConfigGroupFactory, {
 import RangeSliderFactory from 'components/common/range-slider';
 
 import {LAYER_TEXT_CONFIGS} from 'layers/layer-factory';
+import FieldSelectorFactory from '../../common/field-selector';
 
-TextLabelPanelFactory.deps = [RangeSliderFactory, LayerConfigGroupFactory];
-
-export default function TextLabelPanelFactory(RangeSlider, LayerConfigGroup) {
+const TextLabelPanelFactory = (RangeSlider, LayerConfigGroup, FieldSelector) => {
```

Comment:
same here, make it a function instead of const ...
put the deps definition on top of the function declaration for easy read.

Reply:
Done